openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -4,6 +4,7 @@
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
 
 .openassessment {
+  @include direction();
```

Comment:
Looking at the documentation for this mixin (https://github.com/edx/edx-ora2/blob/fd6da7dc8d314f9a79b944c1a3db7b1b6e88e471/openassessment/xblock/static/sass/xb/libs/neat/grid/_direction-context.scss#L7), it looks like an argument should be passed in with it (e.g. right-to-left or left-to-right). 

Is there a reason why this doesn't have an argument? How are you passing in the RTL or LTR setting?